block-esnext/components/Editor.js

```diff
@@ -1,12 +1,17 @@
 /**
  * WordPress dependencies
  */
-const {Component} = wp.element;
-const {Placeholder, withAPIData} = wp.components;
-const {__} = wp.i18n;
+const { Component } = wp.element;
+const {
+	TextControl,
+	SelectControl,
+	Placeholder,
+	withAPIData,
```

Comment:
withAPIData is deprecated & removed https://wordpress.org/gutenberg/handbook/reference/deprecated/#3-7-0

Reply:
Yea, will have to redo it to see what else has moved.

Reply:
so far this one looks out of place & other is `wp-blocks` dependency in styles which is not needed https://github.com/Codeinwp/cf7-gutenberg/blob/master/class-cf7-gutenberg-block-plugin.php#L35.

Also might be good to use `register_block_type()` for registering assets https://wordpress.org/gutenberg/handbook/blocks/writing-your-first-block-type/#enqueuing-block-scripts